### ISSUE TITLE
Choose a local course to insert to save

### ISSUE THREAD

**Issue by MarioPossamato** [Tarnadas/smmdb-client#12]
Would it be possible to, when adding a course to a user's save, have a button that allows a user to choose a local course to insert?  (e.g. be able to choose a zip of tar file containing the course's files.)

**Comment by Rodriguingo5677** [comment]
Yes, that would be a nice feature, because i dump my own packs and it would be nice to be able to play them without having to search them. Thanks for all your work, Tarnadas
